components/31-molecules/form-item/_form-item.scss

```diff
@@ -23,7 +23,7 @@
 
           &:last-child {
             display: flex;
-            align-items: flex-end;
+            align-items: flex-start;
```

Comment:
Why the change?
On closer inspection, this doesn't seem to do anything either way.

Reply:
Kind of weird to display the error message at the bottom of a field. Suppose you have list with 10 checkboxes, you'd have:

```
- [ ]
- [ ]
- [ ]
- [ ]
- [ ]
- [ ]
```

Reply:
I believe this behaviour was by design.

Reply:
@mattiasberlamont, what is the desired behavior here? Should the error message be aligned to the top or bottom of the field?

Reply:
The error message should be aligned to the top. It was designed to be at the top.